tests/setup.ts

```
@@ -1,46 +1,44 @@
-import '@testing-library/jest-dom';
-import { beforeAll, afterAll } from 'vitest';
-import { execSync } from 'child_process';
-import { existsSync, rmSync } from 'fs';
+import "@testing-library/jest-dom";
+import { beforeAll, afterAll } from "vitest";
+import { execSync } from "child_process";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)